runtime/compiler/env/J9ObjectModel.cpp

```diff
@@ -531,23 +531,23 @@ J9::ObjectModel::offsetOfDiscontiguousArraySizeField()
    return compressObjectReferences() ? offsetof(J9IndexableObjectDiscontiguousCompressed, size) : offsetof(J9IndexableObjectDiscontiguousFull, size);
    }
 
-#if defined(TR_TARGET_64BIT)
+#if defined(J9VM_ENV_DATA64)
 uintptr_t
 J9::ObjectModel::offsetOfContiguousDataAddrField()
    {
    return compressObjectReferences()
-		? offsetof(J9IndexableObjectContiguousCompressed, dataAddr)
-		: offsetof(J9IndexableObjectContiguousFull, dataAddr);
+		? offsetof(J9IndexableObjectWithDataAddressContiguousCompressed, dataAddr)
+		: offsetof(J9IndexableObjectWithDataAddressContiguousFull, dataAddr);
    }
 
 uintptr_t
 J9::ObjectModel::offsetOfDiscontiguousDataAddrField()
    {
    return compressObjectReferences()
-		? offsetof(J9IndexableObjectDiscontiguousCompressed, dataAddr)
-		: offsetof(J9IndexableObjectDiscontiguousFull, dataAddr);
+		? offsetof(J9IndexableObjectWithDataAddressDiscontiguousCompressed, dataAddr)
+		: offsetof(J9IndexableObjectWithDataAddressDiscontiguousFull, dataAddr);
    }
-#endif /* TR_TARGET_64BIT */
+#endif /* J9VM_ENV_DATA64 */
```

Comment:
Didn't you agree to fixing these in code you're changing?
It should be `defined(J9VM_ENV_DATA64)` instead of just `J9VM_ENV_DATA64`.